guides/rails_guides/helpers.rb

```diff
@@ -15,7 +15,7 @@ def guide(name, url, options = {}, &block)
     end
 
     def documents_by_section
-      @documents_by_section ||= YAML.load_file(File.expand_path('../../source/documents.yaml', __FILE__))
+      @documents_by_section ||= YAML.load_file(File.expand_path("../../source/#@lang/documents.yaml", __FILE__))
```

Comment:
You should not concatenate the `@lang` variable if it is not set (i.e. guides in English), otherwise we have a useless slash:

~~~ruby
"../../source/#{@lang ? @lang + "/" : ""}documents.yaml"
~~~